packages/salesforcedx-vscode-core/src/commands/util/deployRetrieveCommandlet.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {
+  ApiResult,
+  DeployResult,
+  DeployStatusEnum,
+  SourceClient
+} from '@salesforce/source-deploy-retrieve';
+import { languages, ProgressLocation, window } from 'vscode';
+import { channelService } from '../../channels';
+import { handleDeployRetrieveLibraryDiagnostics } from '../../diagnostics';
+import { nls } from '../../messages';
+import { notificationService } from '../../notifications';
+import { OrgAuthInfo } from '../../util';
+import { LibraryCommandletExecutor } from './libraryCommandlet';
+import { LibraryDeployResultParser } from './libraryDeployResultParser';
+import { outputRetrieveTable } from './retrieveParser';
+
+export class DeployRetrieveLibraryExecutor extends LibraryCommandletExecutor<
```

Comment:
If this only hosts `DeployRetrieveLibraryExecutor` then let's rename the file to `deployRetrieveLibraryExecutor`